dotCMS/src/main/java/com/dotmarketing/portlets/structure/factories/FieldFactory.java

```diff
@@ -113,6 +113,11 @@ public static Field saveField(Field oldField) throws DotHibernateException
             oldField.setFieldContentlet(DataTypes.SYSTEM.toString());
             
         }
+        
+        //The Host or Folder Field and the Tag Field needs to be always indexed (issue #11128)
+        if(oldField.getFieldType().equals(Field.FieldType.HOST_OR_FOLDER.toString()) || oldField.getFieldType().equals(Field.FieldType.TAG.toString())){
```

Comment:
Gonna invert the comparison so the constant come first, but can't remove the toString() in the FieldType since the method getFieldType() returns a String.